frontend/src/content/news.json

```
@@ -1,9 +1,24 @@
 {
   "2022": [
+    {
+      "date": "2022-10-31",
+      "title": "Release 3.1 improves Interaction Partners and Data Overlay",
+      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, now shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Not sure if it's just me but this part: ", now shared between the <i>Interaction Partners</i> and the <i>Map Viewer</I>," while technically true, gives me the impression that this was added in `3.1`. I think this was actually first introduced in this PR: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/723. So I suggest that we can either remove the `now` or the entire quoted part.

```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

---

I'd say it's more debatable - even though is was shared, now we have the reaction overlay which I don't think would have been part of the IP data overlay if it weren't for this work. Anyway, it's safer to remove the `now`,  but I'd go with another suggestion:

```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

---

I like the new suggestion!